ledger/src/blockstore.rs

```diff
@@ -3815,21 +3854,34 @@ where
     Ok(())
 }
 
+// Update the meta for a slot whose parent is known to be connected.
+// Returns a bool indicating whether this slot is now connected which would
+// indicate that it's children are also in need of update.
+fn update_child_from_connected_parent(meta: &mut SlotMeta) -> bool {
+    meta.set_parent_connected(true);
+
+    // If this slot is full, this slot is now connected
```

Comment:
Probably better to describe the actual definition: if a slot is full and its parent is connected, then it is connected.